source/reference/metrics.txt

```diff
@@ -0,0 +1,270 @@
+.. _app-services-metrics:
+
+==============================
+App Services Metrics Reference
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+App Services provides a variety of metrics related to App Services and Device
+Sync. You can access your metrics data with the Admin API's
+:admin-api-endpoint:`getMetrics <tag/metrics/operation/adminGetMetrics>` endpoint.
```

Comment:
nit: i don't think we refer to methods by their operationId elsewhere in the docs. 

i'd just put a plain-text description, smthn like
```rst 
:admin-api-endpoint:`Retrieve App Services metrics <tag/metrics/operation/adminGetMetrics>`
```

(also didn't know we had a `:admin-api-endpoint:` role)

Reply:
It's certainly inconsistent. Usually we refer to the category. I'm referring to a specific endpoint here, so I don't see the benefit of calling it something other than what it is. But I don't feel strongly.

Reply:
but the operationId isn't exposed in the spec (besides the URL path), so don't think it should be used here. recommending: 

```suggestion
:admin-api-endpoint:`Retrieve App Services metrics <tag/metrics/operation/adminGetMetrics>` endpoint.
```

Reply:
Oh, I see what you're saying. Got it.